can-define.js

```diff
@@ -1090,7 +1090,7 @@ define.expando = function(map, prop, value) {
 				type: "can.keys",
 				target: map
 			});
-			if(map._data[prop] !== undefined) {
+			if(map._data.hasOwnProperty(prop)) {
```

Comment:
I'd probably use Object.prototype.hasOwnProperty here ... That will make sure it works if `_data` is Object.create(null)